### PR TITLE
Remove 'murder' from the list of possible eldritch artifact faults

### PR DESCRIPTION
**Pull request by 444Portal**
[BALANCE] [REMOVAL] [SCIENCE]
## About the PR 
Comments out /datum/artifact_fault/murder = 2, from the list of faults the Eldritch Artifacts can roll. That's it. That's the PR.

## Why's this needed? 
Instantly gibbing with no real warning, even if they are funny on paper, aren't fun to have happen to you.

### CODE REVIEW

**Comment by 444Portal** [comment]
I forgot to merge master so it has the commits from my old PR, sorry, I'm pretty dense. shouldn't change anything but artifact_controls.dm however.

**Comment by Laboredih123** [comment]
counterpoint: funny when i die. i meddled with eldritch forces beyond my understanding, funny eldritch force says no

**Comment by Glamurio** [comment]
> counterpoint: funny when i die. i meddled with eldritch forces beyond my understanding, funny eldritch force says no. could lower the chance of no being said.

Counter counterpoint: Funnier if it's not instant. Injects you with a bomb that will gib you. Rush to doctor to be saved or die.

**Comment by MeggalBozale** [comment]
Should definitely go two ways, if artlab can get instant kill weapons they should get weapons that instantly kill themselves and vice versa. I think a cap should be put on both ways for artifacts to keep instant death out of the equation.

File: code/datums/controllers/artifact_controls.dm [hunk 1]
```diff
@@ -476,7 +476,7 @@ var/datum/artifact_controller/artifact_controls
 		'sound/musical_instruments/artifact/Artifact_Eldritch_3.ogg',
 		'sound/musical_instruments/artifact/Artifact_Eldritch_4.ogg')
 	fault_types = list(
-		/datum/artifact_fault/murder = 2,
+//		/datum/artifact_fault/murder = 2,
```
Comment (frawhst) [inline code/datums/controllers/artifact_controls.dm:479]:
```suggestion
```
Just delete, no need to have outdated stuff lying around in comments

**Review by frawhst — COMMENTED** [review]
I'm fine with this as long as you increase the weight of one of the other murdery faults to compensate- add a few points to `poison` or `explode` or something.

**Review by Tarmunora — CHANGES_REQUESTED** [review]
Id rather just change the fault to kill in a way that is borg and/or cloneable, rather than just delete outright

**Comment by MeggalBozale** [comment]
I don't see why making a clone scanner a necessity for artifact lab without being disintegrated should be considered a good idea, let alone any department.

**Comment by 444Portal** [comment]
Going to close this and make a new PR at some point - a bit too tired/busy/whatever and it'll probably go stale. Until then I guess.

**Pull request closed without merging**